metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/utils/EnrichmentEntityDao.java

```diff
@@ -88,9 +91,18 @@
 
   private final MongoClient mongo;
 
+  /**
+   * @deprecated see 
```

Comment:
Why is this marked deprecated? It is just another option of connecting..

Reply:
Mongo connection without authentication is not allowed on production environments. The contructor with the connectionUrl  can be used both for production and development environments (the connection URL mongodb://localhost:27017 doing the same thing).